styx-scheduler-service/src/test/java/com/spotify/styx/state/handlers/FlyteRunnerHandlerTest.java

```diff
@@ -165,7 +155,24 @@ public void shouldNotThrowExceptionIfEvenRouterIsClosed(State state) throws Exce
         .build());
 
     flyteRunnerHandler.transitionInto(runState, eventRouter);
+  }
+
+  @Test
+  public void shouldPollInRunning() throws FlyteRunner.PollingException {
```

Comment:
Please also add another test for `shouldReportRunErrorWhenCatchingExceptionDuringPolling`